frontend/components/Login/Login.tsx

```diff
@@ -3,144 +3,174 @@ import React from "react";
-              >
-                Forgot your password?
-              </a>
+          <div className="entry">
+            <div className="entry-item">
+              <label htmlFor="email">Email Address</label>
+              <input
+                type="text"
+                id="email"
+                name="Email Address"
+                placeholder="Email Address"
+                value=""
+              />
             </div>
-            <input
-              type="password"
-              id="password"
-              name="Password"
-              placeholder="Password"
-            />
-          </div>
-          <div className="working"></div>
+            <div className="entry-item">
+              <div className="password-label">
+                <label htmlFor="password">Password</label>
+                <a
+                  id="forgotPassword"
+                  href="/futurenhsplatform.onmicrosoft.com/B2C_1_logintest/api/CombinedSigninAndSignup/forgotPassword?csrf_token=Q0tQNm51aUlXcXAzamVxNTFuL3c4aVRuWlJQQ3QwSGZ6Yk1KR2YxU1E2WHhUZENta2tZeDVRZzZMbXpMUTF4dHlnQkI3Q2FXWjNNd2tab2ZmWDhYSkE9PTsyMDIwLTA4LTI2VDE0OjA3OjE1Ljg3MjU4MjVaO09lVllQYy8xOW5kdWVnTmhFMEJzMkE9PTt7Ik9yY2hlc3RyYXRpb25TdGVwIjoxfQ==&amp;tx=StateProperties=eyJUSUQiOiJjNTU3YTExOC03MTczLTRkZGQtOGQ5NS0xYTI3YWEzY2U2MjcifQ&amp;p=B2C_1_logintest"
+                >
+                  Forgot your password?
+                </a>
+              </div>
+              <input
+                type="password"
+                id="password"
+                name="Password"
+                placeholder="Password"
+              />
+            </div>
+            <div className="working"></div>
 
-          <div className="buttons">
-            <button id="next" type="submit" form="localAccountForm">
-              Sign in
-            </button>
+            <div className="buttons">
+              <button id="next" type="submit" form="localAccountForm">
+                Sign in
+              </button>
+            </div>
           </div>
-        </div>
-      </form> */}
-    </StyledDiv>
+        </form> */}
+      </StyledDiv>
+      <PasswordDiv>
+        <a
+          id="forgotPassword"
+          href="/futurenhsplatform.onmicrosoft.com/B2C_1_logintest/api/CombinedSigninAndSignup/forgotPassword?csrf_token=Q0tQNm51aUlXcXAzamVxNTFuL3c4aVRuWlJQQ3QwSGZ6Yk1KR2YxU1E2WHhUZENta2tZeDVRZzZMbXpMUTF4dHlnQkI3Q2FXWjNNd2tab2ZmWDhYSkE9PTsyMDIwLTA4LTI2VDE0OjA3OjE1Ljg3MjU4MjVaO09lVllQYy8xOW5kdWVnTmhFMEJzMkE9PTt7Ik9yY2hlc3RyYXRpb25TdGVwIjoxfQ==&amp;tx=StateProperties=eyJUSUQiOiJjNTU3YTExOC03MTczLTRkZGQtOGQ5NS0xYTI3YWEzY2U2MjcifQ&amp;p=B2C_1_logintest"
```

Comment:
This link won't work because the password reset flow needs to be started like a normal OAuth 2 flow. Can you change the URL to:

```suggestion
          href={`${requireEnv('ORIGIN')}/auth/resetpassword`}
```

That would redirect the user to our reset password endpoint, which correctly starts the OAuth 2 flow. You might need to import the `requireEnv` function.

Reply:
I've updated this, but I do get the following error in VSCode:
```Argument of type '"ORIGIN"' is not assignable to parameter of type '"EVENTGRID_TOPIC_ENDPOINT" | "EVENTGRID_TOPIC_KEY" | "PG_URL" | "WORKSPACE_SERVICE_GRAPHQL_ENDPOINT"'.```

Reply:
As I suspected this caused the build to fail, I guess I need to add `ORIGIN` as a type to `requireEnv.js`, would I just do this as:
```const ENV_VAR_NAMES = {
  EVENTGRID_TOPIC_ENDPOINT: 1,
  EVENTGRID_TOPIC_KEY: 1,
  ORIGIN: 1,
  PG_URL: 1,
  WORKSPACE_SERVICE_GRAPHQL_ENDPOINT: 1,
};```? 
Do I need to do anything else?

Reply:
Ah interesting. Yes, you can add `ORIGIN: 1` to that. Nothing else to do.

This type check is to make sure that we don't accidentally try to read environment variables that are not set in production. But `ORIGIN` is set. For `yarn dev` it's pulled in from `.env.development` and for deployments to our cluster it's in the Kubernetes manifests. Looks like we just forgot to add it here to the list.